openedx/core/djangoapps/schedules/content_highlights.py

```diff
@@ -93,6 +91,9 @@ def _get_course_descriptor(course_key):
 
 def _get_course_module(course_descriptor, user):
     # Fake a request to fool parts of the courseware that want to inspect it.
+    from lms.djangoapps.courseware.model_data import FieldDataCache
```

Comment:
My only comment would be to move the `# Fake a request` comment to above the request, and add a separate comment here about why the imports are inside the function.

It would be nice to respect module boundaries a bit better, but I don't have a good suggestion for doing that, at the moment.